src/igvfd/schemas/measurement_set.json

```diff
@@ -302,6 +302,20 @@
                 "igvf_utils": "direct RNA, exome capture"
             }
         },
+        "strand_specificity": {
+            "title": "Strand specificity",
+            "description": "The strand-specificty of the sequencing results within Perturb-seq, scCRISPR screen, TAP-seq, and CERES-seq assays.",
```

Comment:
Small typo, "specificity"

Reply:
Only in the title you mean right? Or should I update it in the description as well?